src/olympia/addons/indexers.py

```diff
@@ -159,7 +159,7 @@ def get_mapping(cls):
                             'average': {'type': 'float', 'index': False}
                         }
                     },
-                    'slug': {'type': 'text'},
+                    'slug': {'type': 'keyword'},
```

Comment:
This needs a reindex unfortunately

Reply:
Yes, but that's fine, it's somewhat backwards-compatible, in the sense that no existing functionality breaks - it's just that slug-based exclusions will not work until the reindex.